include/nihstro/shader_bytecode.h

```diff
@@ -601,11 +601,39 @@ union Instruction {
     } flow_control;
 
     union {
+        const SourceRegister GetSrc1(bool is_inverted) const {
+            if (!is_inverted) {
+                return src1;
+            } else {
+                return src1i;
+            }
+        }
+
+        const SourceRegister GetSrc2(bool is_inverted) const {
+            if (!is_inverted) {
+                return src2;
+            } else {
+                return src2i;
+            }
+        }
+        
+        const SourceRegister GetSrc3(bool is_inverted) const {
+            if (!is_inverted) {
+                return src3;
+            } else {
+                return src3i;
+            }
+        }
+
         BitField<0x00, 0x5, uint32_t> operand_desc_id;
 
         BitField<0x05, 0x5, SourceRegister> src3;
         BitField<0x0a, 0x7, SourceRegister> src2;
         BitField<0x11, 0x7, SourceRegister> src1;
+        
+        BitField<0x05, 0x7, SourceRegister> src3i;
+        BitField<0x0c, 0x5, SourceRegister> src2i;
+        BitField<0x11, 0x7, SourceRegister> src1i;
```

Comment:
That said, GetSrc1 can stay for consistency, but the redundant variable should be removed.